### ISSUE TITLE
Mailtemplate behavior

### ISSUE THREAD

**Issue by zebrahosting** [YetiForceCompany/YetiForceCRM#1678]
Testing with an upgraded setup from 2.0RC via RC5 update to .663

Mail template behavior seems still off.
I have made the same (contact) template for  Email, Records, Module and List. It contains only the 'firstname' variable for testing.

When I open a contact, click the details page and then select to send an email, I see a perfect email with the firstname, opening in RC.

When I do a mass mail from the contact record listing, it brings me to RC, I can only select the Email version of the template but don't get the firstname properly replaced.

Doing the same from RC, by clicking compose, selecting a contact and applying a template, gives no substitution for the first name.

How I should use the list, record and module versions is still not clear.

Bottom line remains that I am not able to send mass emails with the variables working.

**Comment by bpabiszczak** [comment]
At the moment variables work only with the record view, they don't work with any other views [it hasn't been programmed yet]

The system is not yet adapted to mass mail, it will be in the nearest future.
